src/test/lambda/wizards/uploadLambdaWizard.test.ts

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { createWizardTester, WizardTester } from '../../shared/wizards/wizardTestUtils'
+import { UploadLambdaWizard, UploadLambdaWizardState, LambdaFunction } from '../../../lambda/commands/uploadLambda'
+import { makeTemporaryToolkitFolder } from '../../../shared/filesystemUtilities'
+
+describe('UploadLambdaWizard', function () {
+    let tester: WizardTester<UploadLambdaWizardState>
+
+    describe('invoked from command palette', function () {
+        beforeEach(function () {
+            tester = createWizardTester(new UploadLambdaWizard())
+        })
+
+        it('shows all but build prompts', function () {
+            tester.lambda.region.assertShow()
```

Comment:
You can also add a number to assert the order it should be shown in:
```suggestion
            tester.lambda.region.assertShow(1)
```
I'd suggest doing this whenever the order matters. Like `confirmedDeploy` should always be last (fifth)